09-features.md

```diff
@@ -29,7 +29,13 @@ These flags may only be used in the `init` message:
 
 ## Assigned `globalfeatures` flags
 
-There are currently no `globalfeatures` flags.
+The following `globalfeatures` bits are currently assigned by this specification:
+
+|------|-------------------|-------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------|
```

Comment:
markup doesn't render on GitHub, possibly because of this line, which the previous table doesn't have

Reply:
Thanks for the heads up, I fixed the markup and cleaned the tables in the last commit now :-)